crates/biome_js_unicode_table/src/lib.rs

```diff
@@ -6,15 +6,27 @@ mod tables;
 
 pub use crate::bytes::Dispatch;
 
-/// Tests if `c` is a valid start of an identifier
+/// Tests if `c` is a valid start of a js identifier
 #[inline]
-pub fn is_id_start(c: char) -> bool {
+pub fn is_css_id_start(c: char) -> bool {
     c == '_' || c == '$' || ID_Start(c)
 }
 
-/// Tests if `c` is a valid continuation of an identifier.
+/// Tests if `c` is a valid continuation of a js identifier.
 #[inline]
-pub fn is_id_continue(c: char) -> bool {
+pub fn is_css_id_continue(c: char) -> bool {
```

Comment:
I'm unsure how to handle this.
because it's crate for `js`, however it seems it's a bad idea to rename it, because we publish the crate and I don't want to copy for a separate crate.
@ematipico

Reply:
I think we can make this crate more generic and rename it, we shouldn't be afraid. The reason why we publish this crate is because our parsers need it.

We can **deprecate** `biome_js_unicode_table` and have a new crate called `biome_unicode_table`. Obviously, we should this in a separate PR :)